README.md

```diff
@@ -6,56 +6,49 @@
 
 ![logo](https://raw.githubusercontent.com/opentok/accelerator-core/master/tokbox-logo.png)
 
-# OpenTok Accelerator Core
+# OpenTok Accelerator Core JS
 
+OpenTok's `Accelerator Core JS` provides easy integration for OpenTok libraries. Whether you've already built your application using **OpenTok's libraries** or are just starting to use them, `Accelerator Core JS` helps you implement into your application:
 
-```javascript
-otCore.init(options) =>
-  Multi-Party Audio/Video +
-  Text Chat +
-  Screen Sharing +
-  Annotation +
-  Archiving
-```
+  - Multi-Party Audio/Video Communication
+  - Text Chat
+  - Screen Sharing
+  - Annotation
+  - Archiving
+
+##Quickstart
 
-OpenTok Accelerator Core provides a simple way to integrate real-time audio/video into your web application using the OpenTok Platform.  `Core` also integrates with, manages, and provides a single API for the following accelerator packs:
+ To start using audio/video communication with OpenTok's libraries go to [Using accelerator core](#using-accelerator-core). To implement a more granular and complex management for OpenTok's session you can check [Advanced OpenTok Session Management](#advanced-opentok-session-management). 
+ 
+##Using accelerator core
+
+`OpenTok Accelerator Core JS` provides a simple way to integrate real-time audio/video into your web application using the OpenTok Platform. It also integrates with, manages, and provides a single API for the following accelerator packs:
 
  - [Text Chat](https://www.npmjs.com/package/opentok-text-chat)
  - [Screen Sharing](https://www.npmjs.com/package/opentok-screen-sharing)
  - [Annotation](https://www.npmjs.com/package/opentok-annotation)
  - [Archiving](https://www.npmjs.com/package/opentok-archiving)
 
-`Core` also internally maintains the state of your OpenTok session for you.  Calling `otCore.state()` returns an object containing:
-```javascript
-  streams     => All current streams
-  streamMap   => The map of stream ids to publisher/subscriber ids
-  publishers  => All current publishers
-  subscribers => All current subscribers
-  meta        => The count of all current publishers and subscribers by type
-```
-*See an example of the publisher, subscriber, and meta data [below](#pubSubData).*
+##Sample Applications
 
-Sample Applications
--------------------
 There are two sample applications for `Core` .  The [React](https://github.com/opentok/accelerator-core-js/tree/master/react-sample-app) sample application was built with [Create React App](https://github.com/facebookincubator/create-react-app) and uses [webpack](https://webpack.github.io/) to transpile code.  The other [sample application](https://github.com/opentok/accelerator-core-js/tree/master/vanilla-js-sample-app) is built with vanilla JavaScript.
 
-Installation
-------------
+##Installation
 
-```bash
+**bash**
```

Comment:
Remove this line.  It makes things confusing since the following two headers are for specific use cases, while the npm install is universal.